conjure-java-core/src/test/resources/dangerous-name-service.yml

```diff
@@ -52,3 +52,12 @@ services:
             markers:
               - Safe
         returns: string
+      noContext:
+        http: POST /no/context
+        args:
+          requestContext: string
+      context:
+        http: POST /context
+        args:
+          requestContext: string
+        tags: [server-request-context]
```

Comment:
I like people have to add this to get it.

Reply:
I guess we need it for backcompat.

Reply:
Most endpoints shouldn't need the context, so I don't want to generate unnecessary inputs that must be _ignored. When it is needed, it's not necessarily required for every endpoint, so I prefer the tag over a generator option.